src/bcrypt/__init__.py

```diff
@@ -125,6 +126,14 @@ def kdf(password, salt, desired_key_bytes, rounds):
     if rounds < 1:
         raise ValueError("rounds must be 1 or more")
 
+    if rounds < 50 and not ignore_few_rounds:
+        # They probably think bcrypt.kdf()'s rounds parameter is logarithmic,
+        # expecting this value to be slow enough (it probably would be if this
+        # were bcrypt). Emit a warning.
+        sys.stderr.write(("Warning: bcrypt.kdf() called with only {} round(s). "
```

Comment:
I did not know of that library, thanks for the suggestion. I've added another commit to use the library. Is this better?